opentelemetry-sdk/tests/logs/test_handler.py

```diff
@@ -151,7 +153,7 @@ def test_log_record_exception(self):
         log_record = processor.get_log_record(0)
 
         self.assertIsNotNone(log_record)
-        self.assertEqual(log_record.body, "Zero Division Error")
+        self.assertIn("Zero Division Error", log_record.body)
```

Comment:
Could you add an assertEqual check so we're not losing test detail?

Reply:
Does this break the Exception use case even without a formatter? I think we should definitely avoid that if possible.

Reply:
It's caused by the existence of a default log-formatter now, before the output on stdout would be just the string that was passed to `logger.exception`, the default formatter attaches the stacktrace as well:
```text
Zero Division Error
Traceback (most recent call last):
  File "/homeme/dev/opentelemetry-python/opentelemetry-sdk/tests/logs/test_handler.py", line 148, in test_log_record_exception
    raise ZeroDivisionError("division by zero")
ZeroDivisionError: division by zero
```
The original message is always still part of the final result, so I opted to just stick to `assertIn` instead.